ledfx/effects/concentric.py

```diff
@@ -0,0 +1,144 @@
+import numpy as np
+import voluptuous as vol
+from PIL import Image
+
+from ledfx.effects import blur_pixels
+from ledfx.effects.audio import AudioReactiveEffect
+from ledfx.effects.gradient import GradientEffect
+from ledfx.effects.twod import Twod
+
+
+class Concentric(Twod, GradientEffect):
+    """
+    A 2D effect that renders concentric circles expanding from the center
+    on the beat. The colors of the circles are determined by a gradient.
+    """
+
+    NAME = "Concentric"
+    CATEGORY = "2D"
+
+    HIDDEN_KEYS = [
+        "background_brightness",
+        "mirror",
+        "flip",
+        "dump",
+        "gradient_roll",
+        "rotate",
+    ]
+
+    CONFIG_SCHEMA = vol.Schema(
+        {
+            vol.Optional(
+                "frequency_range",
+                description="Frequency range for beat detection",
+                default="Lows (beat+bass)",
+            ): vol.In(list(AudioReactiveEffect.POWER_FUNCS_MAPPING.keys())),
+            vol.Optional(
+                "blur",
+                description="Amount of blur to apply to the final image",
+                default=0.4,
+            ): vol.All(vol.Coerce(float), vol.Range(min=0.0, max=10.0)),
+            vol.Optional(
+                "mirror",
+                description="Mirror the effect",
+                default=False,
+            ): bool,
+            vol.Optional(
+                "invert",
+                description="Inverts the direction of the explosion",
+                default=False,
+            ): bool,
+            vol.Optional(
```

Comment:
Flip is an inherited keyword, don't redefine it. Assuming you just wanted to hide it.